src/octopeer-github/content/EventTracker/WindowResolutionTracker.ts

```diff
@@ -2,25 +2,38 @@
 /**
  * Created by Mathias on 2016-05-26.
  */
-class WindowResolutionTracker {
+class WindowResolutionTracker implements EventTracker {
 
-    /** Private static final for the timeout between logs. */
-    private static get TIMEOUT() { return 500; }
+    /** Public static final for the timeout between logs. */
+    public static get TIMEOUT() { return 500; }
+
+    private static resizeTimer: number;
```

Comment:
TSDoc plox :innocent: 
but remember... copying is bad, mkay (other Trackers have the same field)